src/components/Tokens/TokenDetails/TokenDescription.tsx

```diff
@@ -0,0 +1,156 @@
+import { Trans } from '@lingui/macro'
+import { ChainId } from '@uniswap/sdk-core'
+import Column from 'components/Column'
+import { EtherscanLogo } from 'components/Icons/Etherscan'
+import { Globe } from 'components/Icons/Globe'
+import { TwitterXLogo } from 'components/Icons/TwitterX'
+import CurrencyLogo from 'components/Logo/CurrencyLogo'
+import Row from 'components/Row'
+import { NoInfoAvailable, truncateDescription, TruncateDescriptionButton } from 'components/Tokens/TokenDetails/shared'
+import { useTokenProjectQuery } from 'graphql/data/__generated__/types-and-hooks'
+import { chainIdToBackendName } from 'graphql/data/util'
+import { useCurrency } from 'hooks/Tokens'
+import { useColor } from 'hooks/useColor'
+import useCopyClipboard from 'hooks/useCopyClipboard'
+import { useCallback, useReducer } from 'react'
+import { Copy } from 'react-feather'
+import styled from 'styled-components'
+import { BREAKPOINTS } from 'theme'
+import { ClickableStyle, EllipsisStyle, ExternalLink, ThemedText } from 'theme/components'
+import { opacify } from 'theme/utils'
+import { shortenAddress } from 'utils'
+import { ExplorerDataType, getExplorerLink } from 'utils/getExplorerLink'
+
+const TokenInfoSection = styled(Column)`
+  gap: 12px;
+  width: 100%;
+
+  @media (max-width: ${BREAKPOINTS.lg - 1}px) and (min-width: ${BREAKPOINTS.sm}px) {
```

Comment:
Is the offset to the large breakpoint needed?

Reply:
yes, this matches the useScreenSize hook which is exclusive of the breakpoint value